src/main/java/org/folio/rest/impl/UsersAPI.java

```diff
@@ -315,6 +316,13 @@ private void saveUser(User entity, Map<String, String> okapiHeaders, PostgresCli
                   Arrays.stream(UserType.values()).map(UserType::getTypeName).toList())))));
           return;
         }
+        if (isInvalidUsernameError(reply)) {
```

Comment:
let's add log.warn level message, if it is possible

Reply:
in other places there is not this log, in this case better to add warn log to all other places below

Reply:
Added